### PR TITLE
Bug fix and regression test fix

### PR DESCRIPTION
**Pull request by jsrehak**
Fixes a minor bug in `MaterialProtobuf` listing `do NDA` instead of `do nda`, addes material protobuf file for regression test and updates regression test to use the file.

Regression test output:
visit0000

### CODE REVIEW

File: src/common/problem_definition.cc [hunk 1]
```diff
@@ -121,6 +121,8 @@ void DeclareParameters (dealii::ParameterHandler &local_prm) {
     local_prm.declare_entry ("fuel pin material id file name", "pin_id.txt",
         dealii::Patterns::FileName(),
         "file name for pin material id map for pin-resolved calculations");
+    local_prm.declare_entry("material id file name map", "",
+                            dealii::Patterns::Map(dealii::Patterns::Integer(),dealii::Patterns::FileName(dealii::Patterns::FileName::input)));
```
Comment (weixiong-zheng-berkeley) [inline src/common/problem_definition.cc]: Break Line 125, it's too long

File: reg-test/t-ep [hunk 2]
```diff
@@ -27,29 +27,10 @@ set finite element polynomial degree         = 1
...
-set values                                   = 1.0, 1.0
-end
-
-subsection one-group sigma_s
-set values                                   = 0.15, 0.15
-end
-
-subsection one-group Q
-set values                                   = 0.
+set material id file name map                = 1: t-ep.material, 2: t-ep.material
```
Comment (weixiong-zheng-berkeley) [inline reg-test/t-ep]: This is neat!

File: src/common/problem_definition.cc (hunk 1 above)
Comment (weixiong-zheng-berkeley) [reply src/common/problem_definition.cc]: You can also move `dealii::xxxx` towards left until the fourth indent space, read Google Style if you want to.

Comment (jsrehak) [reply src/common/problem_definition.cc]: I meant to fix that, thanks.

**Pull request merged**